objects.js

```diff
@@ -8520,6 +8523,48 @@ WatcherMorph.prototype.userMenu = function () {
     return menu;
 };
 
+WatcherMorph.prototype.valueExporter = function (format) {
+    var myself = this,
+        value = this.currentValue,
+        contents,
+        format = format || 'plain',
+        ide = myself.parentThatIsA(IDE_Morph);
+
+    return function () {
+        switch (format) {
+            case 'plain':
+                contents = value instanceof List ? 
+                    value.asArray().join(', ') : 
```

Comment:
Yeah, but that's just a matter of taste and particular use cases.

CSV stands for **Comma** Separated Values, so it stands to reason that default separators should be commas.

One-dimensional lists can be thought of as single-row tables (thus comma-separated) or single-column tables (thus newline-separated). To me, it makes much more sense to think of them as rows, but again it's just a matter of your particular use case.